### DISCUSSION TITLE
Better access control of existing models

### DISCUSSION THREAD

**Discussion by 0rangeMan** [open-webui/open-webui#6849] [Ideas]
Is it possible to have a better model management? e.g. model management to user level or maybe user group level?
Currently it is only white listing for all users.

**Comment by flefevre** [comment]
I agree with you. Perhaps by adding the feature 'team' it could be taken also the access of the different models? Eventually a link between 'open webui' team' and 'litellm team' could be done? In litellm you can define team with access to specific list of model thanks to virtual keys. Perhaps those token keys could be set in 'open webui' ?

**Comment by ill-yes** [comment]
Would be awesome to have usergroups! 🚀
